ui/app/templates/vault/cluster/access/oidc/clients/client.hbs

```diff
@@ -3,7 +3,7 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-{{#if this.showHeader}}
```

Comment:
this is the refactor removing the showHeader getter from the OidcClientController3